packages/poseidon-cipher/package.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "@zk-kit/poseidon-cipher",
+    "version": "0.1.0",
+    "description": "Poseidon encryption and decryption in TypeScript.",
+    "license": "MIT",
+    "keywords": [
+        "typescript",
+        "poseidon",
+        "cryptography",
+        "encryption",
+        "decryption",
+        "ciphertext",
+        "plaintext"
+    ],
+    "iife": "dist/index.js",
+    "unpkg": "dist/index.min.js",
+    "jsdelivr": "dist/index.min.js",
+    "main": "dist/index.node.js",
+    "exports": {
+        "import": "./dist/index.mjs",
+        "require": "./dist/index.node.js",
+        "types": "./dist/types/index.d.ts"
+    },
+    "types": "dist/types/index.d.ts",
+    "files": [
+        "dist/",
+        "src/",
+        "README.md",
+        "LICENSE"
+    ],
+    "repository": "git@github.com:privacy-scaling-explorations/zk-kit.git",
+    "homepage": "https://github.com/privacy-scaling-explorations/zk-kit/tree/main/packages/poseidon-cipher",
+    "scripts": {
+        "build:watch": "rollup -c rollup.config.ts -w --configPlugin typescript",
+        "build": "rimraf dist && rollup -c rollup.config.ts --configPlugin typescript",
```

Comment:
This needs to be updated. It should be like: https://github.com/privacy-scaling-explorations/zk-kit/blob/a2863c71dfdc5ac5790b80760c9d601c484f6140/packages/baby-jubjub/package.json#L28

Reply:
good catch, thank you